.github/workflows/jlab-a11y-test.yml

```diff
@@ -0,0 +1,76 @@
+name: Run accessibility tests on JupyterLab
+
+on: [workflow_dispatch, push, pull_request]
+
+defaults:
+  run:
+    working-directory: ./testing/jupyterlab
+
+jobs:
+  jlab-a11y-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository 🛎
+        uses: actions/checkout@v3
+
+      - name: Set up Python 🐍
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      - name: Set up Node 🧶
+        uses: actions/setup-node@v3
+        with:
+          node-version: "14.x"
+
+      - name: Cache conda 🧠
+        uses: actions/cache@v2
+        env:
+          # Increase this value to reset cache if environment.yml has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
+            hashFiles('environment.yml') }}
+
+      - name: Install miniconda 🐍
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          use-mamba: true
+          environment-file: ./testing/jupyterlab/environment.yml
+          activate-environment: a11y-tests
+          use-only-tar-bz2: true
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache yarn 🧠
+        uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install node dependencies 🧶
+        run: |
+          yarn install
+          npx playwright install
+
+      - name: Run tests  ✅
+        run: yarn test
+
+      - name: Upload testing results 📤
+        uses: actions/upload-artifact@v3
+        # using default retention policy = 90 days
+        with:
+          name: jupyterlab-a11y-tests
+          path: |
+            jupyter-ally-build/jupyter-axe/test-results/*.html
+            jupyter-ally-build/jupyter-axe/test-results/**/*.html
+            jupyter-ally-build/jupyter-axe/test-results/*.json
+            jupyter-ally-build/jupyter-axe/test-results/**/*.json
```

Comment:
I am 💯 per cent sure this is the wrong path - but since I could not run these locally I could not verify. @gabalafou  can you please confirm the correct path where the tests results are stored?

Reply:
Updated these paths, but please see my inline review comment.